package-lock.json

```diff
@@ -7834,10 +7844,29 @@
                 "request": "^2.83.0",
                 "request-promise": "^4.2.2",
                 "simple-git": "~1.92.0",
-                "vscode-azureextensionui": "^0.26.2",
+                "vscode-azureextensionui": "^0.27.0",
                 "vscode-azurekudu": "^0.1.9",
                 "vscode-nls": "^4.0.0",
                 "websocket": "^1.0.25"
+            },
+            "dependencies": {
+                "vscode-azureextensionui": {
```

Comment:
You should update the ui package (and address the breaking changes) at the same time so you don't have multiple different ui packages